code/modules/mob/living/silicon/say.dm

```diff
@@ -10,24 +10,41 @@
 	if(issilicon(src))
 		var/mob/living/silicon/S = src
 		desig = trim_left(S.designation + " " + S.job)
-	var/message_a = say_quote(message)
-	var/rendered = "Robotic Talk, <span class='name'>[name]</span> <span class='message'>[message_a]</span>"
+	var/large_message_a = say_quote(message, list("robot big"))
+	var/message_a = say_quote(message, list("robot"))
 	for(var/mob/M in GLOB.player_list)
 		if(M.binarycheck())
 			if(isAI(M))
-				var/renderedAI = "<span class='srt_radio binarysay'>Robotic Talk, <a href='?src=[REF(M)];track=[html_encode(name)]'><span class='name'>[name] ([desig])</span></a> <span class='message'>[message_a]</span></span>"
-				to_chat(M, renderedAI)
+				var/source = FALSE
```

Comment:
nitpick. instead of `var/source`, let it be named `var/loud`
do the same to other variables